src/components/20-molecules/list/child.scss

```diff
@@ -0,0 +1,40 @@
+axa-list {
+    font-size: 16px;
```

Comment:
try to use variables from the variables.scss files

Reply:
Good point thx.